site/content/docs/5.2/components/modal.md

```diff
@@ -498,6 +512,13 @@ Have a bunch of buttons that all trigger the same modal with slightly different
 
 Below is a live demo followed by example HTML and JavaScript. For more information, [read the modal events docs](#events) for details on `relatedTarget`.
 
+<!-- Boosted mod : design callout -->
+{{< ods-incompatibility-alert >}}
```

Comment:
Mainly because of the content of the modal.

Reply:
New message:
These variants of modals with forms should not be used because they do not respect the Orange Design System specifications. 
For the correct design, please refer to [Modals](https://system.design.orange.com/0c1af118d/p/16d9f3-modals/b/774d3d) on the Orange Design System website.